AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -112,4 +102,11 @@ - (void)removeDelegate:(id<MSAuthTokenContextDelegate>)delegate {
   }
 }
 
+- (void)cacheAuthToken {
```

Comment:
What's the purpose of this method? Seems it contains current `authToken` and `homeAccountId` that can be handled in `setAuthToken` I guess.

Reply:
Yes, it can be replaced by `setAuthToken`.
The purposes:
- Harmonize with Android
- wrap almost operations with token storage into context